app/assets/javascripts/galleries/uploader.js

```diff
@@ -129,3 +130,8 @@ function deleteUnusedIcons(keys) {
     $.authenticatedPost('/api/v1/icons/s3_delete', {s3_key: key});
   });
 }
+
+function isPolicyExpired(responseText) {
+  if (responseText.includes("Invalid according to Policy: Policy expired.")) { return true; }
+  return responseText.includes("Idle connections will be closed.");
```

Comment:
```suggestion
  return responseText.includes("Invalid according to Policy: Policy expired.") || responseText.includes("Idle connections will be closed.");
```
(can be split across lines if we have to, but I forget what JavaScript's rules are for continuation)

Reply:
Yeah I split it like that because the line was long and I was extracting it anyway :+1:

Reply:
```suggestion
  return responseText.includes("Invalid according to Policy: Policy expired.")
    || responseText.includes("Idle connections will be closed.");
```
then? having an extra if takes a lot more time for me to parse here